utility/src/main/java/org/oppia/android/util/profile/NameValidator.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.android.util.profile
+
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Utility to validate profile names */
+@Singleton
+class NameValidator @Inject constructor() {
```

Comment:
Nit: ProfileNameValidator would probably be clearer since 'name' is a bit generic in itself.